.github/CONTRIBUTING.md

```diff
@@ -1,9 +0,0 @@
-## Issues
```

Comment:
This file is used by github - see https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors and https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/encouraging-helpful-contributions-to-your-project-with-labels.

From a quick at the github documentation, rst is not mentioned anywhere, do we know if github can parse the `rst` format? Maybe simply add a link to a markdown file located at the right place, as there are alternatives. Using the https://github.com/LumiSpy/.github would have the advantage of keeping this repository tidy.

Reply:
Should be doable with the Markdown extension for Sphinx: https://www.sphinx-doc.org/en/master/usage/markdown.html

Then the files in the main folder would remain .md and could still be integrated in the docs. I'll check that out later today.

Reply:
But, `rst` would be supported by GitHub as well: https://docs.github.com/en/repositories/working-with-files/using-files/working-with-non-code-files#rendering-differences-in-prose-documents

Reply:
OK, that would be brilliant! Then, once this is merged, github should show us a notification when making a new PR, because it should detect that the contributing guide has changed.